tests/test_free_fleet.py

```
@@ -0,0 +1,112 @@
+"""‏hard $0 — הדרישה החשובה ביותר של נדב, נאכפת מבנית.
+
```

Ruff (0.16.3)

[error] Contains control characters that can permit obfuscated code

(PLE2502)